research_daps/orms/glass.py

```diff
@@ -0,0 +1,202 @@
+# type: ignore
+"""Glass AI ORMs
+
+Assumptions:
+- Organisation names and URL's do not change
+- Sectors do change [TODO]
+- Addresses do change [TODO]
+
+"""
+from sqlalchemy import Table, Column, Integer, String, ForeignKey, Date, Boolean
+from sqlalchemy.orm import relationship, backref
+from research_daps import declarative_base
+
+Base = declarative_base()
+
+# organisation_address = Table(
+#     "organisation_address",
+#     Base.metadata,
+#     Column("org_id", Integer, ForeignKey("organisation.org_id")),
+#     Column("address_id", Integer, ForeignKey("address.address_id")),
+#     # TODO: Active? boolean column indicating whether address is in latest data dump
+#     # TODO: Rank Integer column indicating address rank
+#     # TODO: Date Date column indicating date of latest dump with this relationship
+# )
+class OrganisationAddress(Base):
+    """Association object between organisations and addresses"""
+
+    org_id = Column(Integer, ForeignKey("organisation.org_id"), primary_key=True)
```

Comment:
Make sure to set `autoincrement=False` else prepare yourself for a world of pain

Reply:
(do this for all pks on Integer columns in this ORM)